src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { styled } from '@mui/material';
+
+import { StyledFlexTopCenter, variables, theme } from 'shared/styles';
+import { SelectController } from 'shared/components/FormComponents';
+
+export const StyledSummaryRow = styled(StyledFlexTopCenter)`
+  gap: 0.4rem;
+  padding: ${theme.spacing(0.8, 1.2)};
+
+  & > * {
```

Comment:
is it possiple to specify with more restricted rules?

Reply:
done